src/mgr/Action.cc

```diff
@@ -111,12 +111,35 @@ Mgr::Action::fillEntry(StoreEntry* entry, bool writeHttpHeader)
 
         entry->replaceHttpReply(rep);
     }
-
+    if (!aggregatable())
+        openKidSection(entry, is_yaml());
```

Comment:
AFAICT, this new condition is wrong and should be removed. Not all actions that contain "aggregatable" data need to open a kid-produced document. Some aggregatable() actions may have no kid-produced documents at all.

```suggestion

```

... plus restoring the old line; I cannot do that via a GitHub `suggestion` block.

Reply:
Huh? I read the opposite: if the action can**not** produce aggregatable data, the infrastructure will do the framing for it.
The assumption is that if data _is_ aggregatable, the action has been touched and knows what to do

Reply:
My bad: I made a mistake either in requesting this change or while explaining _why_ this change should be made. I am resolving this change request for now. I may come back to it later after I have the time to study this code timing.

Thank you for explaining your logic.